src/assets/data/locals.json

```
@@ -166,6 +166,19 @@
     "imagem": "assets/images/pelourinho-salvador.jpg",
     "titulo": "Pelourinho",
     "descricao": "Inicialmente lugar de punição de escravos, o Pelourinho reflete a própria história de Salvador. Nas últimas décadas, o Pelourinho passou a atrair artistas de todos os gêneros: cinema, música, pintura, tornando-o um importante centro cultural de Salvador."
+  },
+  {
+    "uf": "MA",
+    "cidade": "São Luís",
+    "imagem": "assets/images/casa-das-tulhas-centro-historico-sao-luis.jpg",
```

Faltou só uma letrinha aqui (ou sobrou no nome da imagem rs)
```suggestion
    "imagem": "assets/images/casa-das-trulhas-centro-historico-sao-luis.jpg",
```

---

realmente, na verdade o nome do arquivo está errado. atualizei com o nome correto.